config/settings.yml

```diff
@@ -1543,6 +1543,10 @@ form526_backup:
     access_key_id: ~
     secret_access_key: ~
 
+forms_api_benefits_intake:
+  api_key: ~
+  url: https://sandbox-api.va.gov/services/vba_documents/v1/
```

Comment:
Is this necessary to include here? I got a little lost in yours and Rebecca's convo

Reply:
That's a good point. Since a lot of other teams include them, I think the key benefit is acknowledging the settings that exist, even if they are injected elsewhere (manifests, devops). For example, if you wanted to set up the local.settings.yml I think it's nice to know what you can configure.

Just my two thoughts!

Reply:
Makes sense to me